app/jobs/organization_notifier_job.rb

```diff
@@ -0,0 +1,12 @@
+class OrganizationNotifierJob < ActiveJob::Base
```

Comment:
It'd be great to explain what it notifies about and who without having to read the code. I'd be much faster for readers to better understand what it does.